scripts/db_migrator.py

```diff
@@ -102,6 +107,41 @@ def migrate_interface_table(self):
                 self.configDB.set_entry(table, key[0], data[key])
                 if_db.append(key[0])
 
+    def migrate_intf_table(self):
```

Comment:
Suggest to rename to migrate_app_intf_table, since there is already migrate_interface_table function.

Reply:
yes, but the table name here is 'intf', so that's why chose the name like that.